pkg/apis/v1alpha1/condition.go

```diff
@@ -0,0 +1,29 @@
+package v1alpha1
```

Comment:
could we rename the file as condition_types.go?

Reply:
sure, fixed.